tests/nlu/data_augmentation/test_augmenter.py

```diff
@@ -0,0 +1,332 @@
+import os
+from typing import Callable
+
+from _pytest.pytester import RunResult, Testdir
+from rasa.nlu.components import Component
+from rasa.nlu.constants import TOKENS_NAMES
+from rasa.shared.constants import LATEST_TRAINING_DATA_FORMAT_VERSION
+from rasa.shared.nlu.training_data.message import Message
+import rasa.shared.nlu.training_data.loading
+import rasa.shared.utils.io
+from rasa.shared.nlu.constants import (
+    INTENT,
+    TEXT,
+    VOCABULARY,
+)
+
+
+def test_augmenter_create_tokenizer(run: Callable[..., RunResult],):
+    from rasa.nlu.data_augmentation.augmenter import _create_tokenizer_from_config
+
+    data_root = os.path.dirname(
+        os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+    )
+    config_path = os.path.join(data_root, "data/test_nlu_paraphrasing/config.yml")
+
+    tokenizer = _create_tokenizer_from_config(config_path=config_path)
+
+    assert isinstance(tokenizer, Component)
```

Comment:
We should assert that it's an instance of `Tokenizer`

Reply:
But `_create_tokenizer_from_config` returns a `Component` (b/c its created via `ComponentBuilder().create_component(...))`. 

Is there a better way to create it?

Reply:
The return type is `Component` of that function because it's a general purpose function for any component. `Tokenizer` is a child class of `Component` and that's the class that you expect `tokenizer` to be an instance of. Does that clarify?